sab-run.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Scripture App Builder
+# Command line app build and launch wrapper
+# Use sab.sh -? or see the document 'Building Apps' for more information
+
+# echo "Running scripture-app-builder in:"
+# echo $( pwd )
+# echo $( /app/jre/bin/java -version )
+# echo $( /app/jdk17.0.11/bin/javac -version )
+# echo $( python3 --version )
+# echo $( which python3 )
+# echo $( python3 -m aeneas.diagnostics )
+# echo " "
+# echo "$@"
+# echo " "
+exec /app/jre/bin/java --module-path "/app/lib/sdk" --add-modules javafx.web,javafx.fxml,javafx.swing,javafx.media --add-opens=javafx.fxml/javafx.fxml=ALL-UNNAMED -jar /app/scripture-app-builder/bin/scripture-app-builder.jar $@
```

Comment:
```suggestion
exec /app/jre/bin/java --module-path "/app/lib/sdk" --add-modules javafx.web,javafx.fxml,javafx.swing,javafx.media --add-opens=javafx.fxml/javafx.fxml=ALL-UNNAMED -jar /app/scripture-app-builder/bin/scripture-app-builder.jar "$@"
```

Reply:
This should not be quoted since it could be a series of command-line arguments.

Reply:
no. 

```
@      Expands to the positional parameters, starting from one. [...] When the expansion occurs within double quotes, each parameter expands to a separate word.  That
is, "$@" is equivalent to "$1" "$2" ...
```

Reply:
Thanks for the clarification.  I will try that out.

Reply:
Changed.  This worked.  Thanks!